SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverRPC.cpp

```diff
@@ -381,12 +381,15 @@ bool FSpatialNetDriverRPC::ApplyRPC(Worker_EntityId EntityId, const FRPCPayload&
 	if (TargetObject == nullptr)
 	{
 		const TWeakObjectPtr<UObject> ActorReceivingRPC = NetDriver.PackageMap->GetObjectFromEntityId(EntityId);
-		AActor* Actor = CastChecked<AActor>(ActorReceivingRPC.Get());
-		checkf(Actor != nullptr, TEXT("Receiving actor should have been checked in CanReceiveRPC"));
-		UE_LOG(LogSpatialNetDriverRPC, Error,
-			   TEXT("Failed to execute RPC on Actor %s (Entity %llu)'s Subobject %i because the Subobject is null"), *Actor->GetName(),
-			   EntityId, RPCData.Offset);
-
+		if (ActorReceivingRPC.Get())
```

Comment:
Nit: you could avoid indenting all this code with an early out but some prefer one-code-path also. I don't mind either, less indentation is probably nicer imo though :D

Reply:
and less lines of code touched - which is very useful when using "git blame"

Reply:
Thank you, will keep this in mind for the future :) !